test/utils/autoload/lsp/test.vim

```diff
@@ -31,6 +33,23 @@ function! lsp#test#openproject(name, options) abort
             \ lsp#callbag#take(1),
             \ lsp#callbag#toList(),
             \ ).wait({ 'timeout': 10000, 'sleep': 100 })
```

Comment:
should we also do the same here as golang

open -> close -> wait -> bwipeout

im more than happy to remove rust then we don't need to worry about it.

Reply:
I removed unused test helper for Rust project as much as possible at ca86fb4. Whole project directory and helpers could not be removed because `lsp#test#projectdir('rust')` is used by other test case.